### ISSUE TITLE
LPIPS

### ISSUE THREAD

**Issue by korawat-tanwisuth** [HelenMao/MSGAN#18]
Hi @HelenMao,

For LPIPS, I am assuming you use Alexnet and version 0.0. For NDB, could you please share the hyperparameters (other than K) to the ImageBatchProvider, NDB, and evaluate function? Thanks!

**Comment by HelenMao** [comment]
Hi, we use the original image for calculation and adopt the default setting of the LPIPS codes.
For NDB, we also use the default setting of codes provided by NDB's author.
What do the ImageBatchProvider and evaluation function mean?

**Comment by korawat-tanwisuth** [comment]
Hi @HelenMao 

Thank you for your prompt response. In issue #14, you mentioned that you use the eval code as the reference: https://github.com/eitanrich/gans-n-gmms/blob/b61d6abb69caa80d5aa84a72434784d6460ba10c/mfa_eval_celeba.py. I am referring to the imagebatchprovider function in that file. Basically, it has some hyperparameters that do some pre-processing to the image. Also, in the NDB function, it has some hyperparameter like whitening (normalizing the data). I am not quite sure what to set those too. Thank you again!

**Comment by HelenMao** [comment]
We only use the NDB function, i.e., Line 76 ndb.NDB(training_data=train_samples, number_of_bins=num_bins, whitening=True, cache_folder=args.ndb_dir) as reference. 
I did not use the imagebatchprovider, and directly construct the generated images into the .npy file and load it for calculation.
Since the NDB codes have been modified many times by its authors, we used a very early version of the NDB codes in the CVPR version.
Other hyperparameters such as whitening=True are adopted using the default setting.
